src/Domain/Creators/Brick.hpp

```diff
@@ -79,17 +81,46 @@ class Brick : public DomainCreator<3> {
         "The time dependence of the moving mesh domain."};
   };
 
-  using options =
-      tmpl::list<LowerBound, UpperBound, IsPeriodicIn, InitialRefinement,
-                 InitialGridPoints, TimeDependence>;
+  template <typename BoundaryConditionsBase>
+  struct BoundaryCondition {
+    static std::string name() noexcept { return "BoundaryCondition"; }
+    static constexpr Options::String help =
+        "The boundary condition to impose on all sides.";
```

Comment:
Since it's fairly straight-forward to add support for different boundary conditions on either side, how about we make these a bunch of `good first issue`s (also for the other domain creators that have "not yet implemented" asserts)? :)

Reply:
Added #2871